### PR TITLE
update tagline

### PR DESCRIPTION
**Pull request by Sertonix**
This updates the tagline to match the voted one: "A Community-led Hyper-Hackable Text Editor"

### CODE REVIEW

**Comment by DeeDeeG** [comment]
Not sure if this was intentional as a cheeky "editor of the future" flavor text, but yeah, I think consistency is good, so we should pick one or the other.

Would like to hear @mauricioszabo's thoughts, who is the person who committed the "22nd century" bit.

But it is just flavor text, so whatever we choose is just for branding/for fun.

**Comment by Daeraxa** [comment]
Ultimately we should pick whatever it is that our tagline is going to be - see https://github.com/orgs/pulsar-edit/discussions/50

**Comment by mauricioszabo** [comment]
I would choose `A hackable text editor for the future` for now, until we decide on an official tagline :D

I actually made the commit just because I wanted to test what difference that would make, but to be honest, I forgot about it :)

**Comment by DeeDeeG** [comment]
Thanks @mauricioszabo.

(Sounds like we can hold off committing any changes to this until we have an agreed tagline? If that never happens, then I support changing it to "21st" eventually, if only so search-and-replace doesn't miss it!)

**Comment by confused-Techie** [comment]
@Sertonix Our [poll](https://discord.com/channels/992103415163396136/1040070150298292325/1040111779780493413) has confirmed our final tagline, so if you could please get this data in line with what the community has voted on:

"A Community-led Hyper-Hackable Text Editor"

**Comment by confused-Techie** [comment]
Thanks for getting those requested changes in!

Thanks for the contribution!

**Review by confused-Techie — APPROVED** [review]

File: packages/about/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ View useful information about your Atom installation.
 
 ![About Atom](https://cloud.githubusercontent.com/assets/16760489/19395499/69bbb780-922d-11e6-9779-2b8327027ea5.png)
 
-This is a package for [Atom](https://atom.io), a hackable text editor for the 21st Century.
+This is a package for [Atom](https://atom.io), a community-led hyper-hackable text editor
```
Comment (Daeraxa) [inline packages/about/README.md:8]: This whole file probably needs to be changed at some point with rebranding.

Comment (Sertonix) [reply packages/about/README.md:8]: Yes I know but that is out of scope for this pull request.

Comment (confused-Techie) [reply packages/about/README.md:8]: > This whole file probably needs to be changed at some point with rebranding.

Will probably need to earmark it for later, or make an issue for it.

Comment (Daeraxa) [reply packages/about/README.md:8]: I agree, just wanted to call attention to it for later as *technically* Atom isn't the "community-led......editor"

Comment (confused-Techie) [reply packages/about/README.md:8]: > I agree, just wanted to call attention to it for later as _technically_ Atom isn't the "community-led......editor"

Actually fantastic point. Think I might just merge this one, and immediately make a PR to rebrand these documents.

**Comment by confused-Techie** [comment]
Considering any issues have been addressed, and this otherwise looks good I'll go ahead and merge. Thanks for the contribution!

**Pull request merged**